packages/api/src/models/taskModel.js

```diff
@@ -31,7 +31,7 @@ class TaskModel extends BaseModel {
   static get jsonSchema() {
     return {
       type: 'object',
-      required: ['task_id', 'due_date', 'type'],
+      required: ['due_date', 'type'],
```

Comment:
why is task_id no longer required?

Reply:
it cant be required for posts (because it doesn't exist) it follows the same pattern as all other models